llvm/lib/CodeGen/SelectionDAG/SelectionDAGBuilder.cpp

```diff
@@ -2119,7 +2060,9 @@ static void findUnwindDestinations(
       // personalities.
       UnwindDests.emplace_back(FuncInfo.getMBB(EHPadBB), Prob);
       UnwindDests.back().first->setIsEHScopeEntry();
-      UnwindDests.back().first->setIsEHFuncletEntry();
+      // In Wasm, EH scopes are not funclets
```

Comment:
Maybe the comment on line 2059 above should be updated too.

Reply:
Done